File: Facebook.NET/Contracts/GraphAPI.Cover.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed class Cover
        {
            [DataMember(Name = "cover_id")]
            public string ID { get; set; }

            [DataMember(Name = "source")]
            public string Source { get; set; }

            [DataMember(Name = "offset_x")]
            public double? OffsetX { get; set; }

            [DataMember(Name = "offset_y")]
            public double? OffsetY { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Cover.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Cover
            <DataMember(Name:="cover_id")>
            Public Property ID() As String
            <DataMember(Name:="source")>
            Public Property Source() As String
            <DataMember(Name:="offset_x")>
            Public Property OffsetX() As System.Nullable(Of Double)
            <DataMember(Name:="offset_y")>
            Public Property OffsetY() As System.Nullable(Of Double)
        End Class
    End Class
End Namespace
